das/daser.go

```diff
@@ -14,7 +14,6 @@ import (
 var log = logging.Logger("das")
 
 // DASer continuously validates availability of data committed to headers.
-// TODO(@Wondertan): Start and Stop is better be thread-safe.
```

Comment:
Actually no, we should never call Start and Stop from different routines and we are doing so, thus this todo is not relevant